examples/chef/linux/main.cpp

```diff
@@ -49,6 +57,10 @@ int main(int argc, char * argv[])
     cmd_app_server_init();
 #endif
 
+#if CHIP_CONFIG_ENABLE_ICD_CIP
+    DeviceLayer::SystemLayer().StartTimer(chip::System::Clock::Milliseconds32(CHIP_CONFIG_ICD_IDLE_MODE_DURATION_SEC*1000), notifyIcdActive, nullptr);
```

Comment:
Having this timer here feels 100% wrong. Why is this in main? Why does it notify forever?

Reply:
This is removed now.